### DISCUSSION TITLE
Install/Config coc using Lazy

### DISCUSSION THREAD

**Discussion by ghost** [neoclide/coc.nvim#4963] [Q&A]
Greetings everyone,

I would like to know how I can install and configure coc in NeoVim using the Lazy plugin manager, I tried to follow the example described in this [gits](https://gist.github.com/dinhmai74/4fc774dfd69750ce9ed706620a5082a9), the coc is simply installed, but does not work.

**Comment by fannheyward** [comment]
What do you mean `does not work`? Check `:CocInfo`.

**Reply by ghost** [reply]
Problem already solved!

**Reply by IsaacCooke** [reply]
How was it solved? I'm having the same issue myself.

**Reply by by-justin** [reply]
If it's solved that easy, maybe you shouldn't open up a issue; if it is not that easy, maybe you could disclose a bit more about how you solved it. Otherwise you are just taking up space on google search results with meaningless information.
